commit 756a3512ce74b93953ced1d99e063b496f9a920b
Author: Kefu Chai <kchai@redhat.com>
Date:   Mon Jun 7 16:31:11 2021 +0800

    *: always include <filesystem>
    
    since there is no need to be compatible with GCC older than GCC-8, so
    there is no need to use <experimental/filesystem> as an alternative of
    <filesystem> anymore.
    
    Signed-off-by: Kefu Chai <kchai@redhat.com>

diff --git a/src/mgr/PyModuleRegistry.cc b/src/mgr/PyModuleRegistry.cc
index 79c42653620..f5f5008023f 100644
--- a/src/mgr/PyModuleRegistry.cc
+++ b/src/mgr/PyModuleRegistry.cc
@@ -13,15 +13,7 @@
 
 #include "PyModuleRegistry.h"
 
-#if __has_include(<filesystem>)
 #include <filesystem>
-namespace fs = std::filesystem;
-#elif __has_include(<experimental/filesystem>)
-#include <experimental/filesystem>
-namespace fs = std::experimental::filesystem;
-#else
-#error std::filesystem not available!
-#endif
 
 #include "include/stringify.h"
 #include "common/errno.h"
@@ -42,6 +34,8 @@ namespace fs = std::experimental::filesystem;
 #undef dout_prefix
 #define dout_prefix *_dout << "mgr[py] "
 
+namespace fs = std::filesystem;
+
 std::set<std::string> obsolete_modules = {
   "orchestrator_cli",
 };
